_posts/en/newsletters/2024-08-23-newsletter.md

```diff
@@ -0,0 +1,188 @@
+---
+title: 'Bitcoin Optech Newsletter #317'
+permalink: /en/newsletters/2024/08/23/
+name: 2024-08-23-newsletter
+slug: 2024-08-23-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **Simple (but imperfect) anti-exfiltration protocol:** developer
+  Moonsettler [posted][moonsettler exfil1] to Delving Bitcoin to describe an
+  [anti-exfiltration][topic exfiltration-resistant signing] protocol.
+  The same protocol has been described before (see Newsletters
+  [#87][news87 exfil] and [#88][news88 exfil]), with Pieter Wuille
+  [citing][wuille exfil1] the earliest known description of the
+  technique for anti-exfil being a [2014 post][maxwell exfil] by Gregory
+  Maxwell.
+
+  The protocol uses the sign-to-contract protocol to allow a software
+  wallet to contribute entropy to the nonce selected by a hardware
+  signing device in a way that allows the software wallet to later
+  verify the entropy was used.  Sign-to-contract is a variation on
+  [pay-to-contract][topic p2c]: in pay-to-contract, the receiver's
+  public key is tweaked; in sign-to-contract, the spender's signature
+  nonce is tweaked.
+
+  The advantage of this protocol, compared to the protocol implemented
+  for BitBox02 and Jade hardware signing devices (see [Newsletter
+  #136][news136 exfil]), is that it only requires one round trip of
+  communication between the software wallet and the hardware signing
+  device.  That one round trip can be combined with the other steps
+  necessary to sign for a single-sig or scripted multisig transaction,
+  meaning the technique doesn't affect user workflows.  The currently
+  deployed technique, which is also based on sign-to-contract, requires
+  two round trips; that's more than required for most users today,
+  although multiple round trips may be required for users who upgrade to
+  using [scriptless multisignatures][topic multisignature] and
+  [scriptless threshold signatures][topic threshold signature].  For
+  users who connect their signing devices directly to their computers or
+  who use an interactive wireless communication protocol like Bluetooth,
+  the number of round trips doesn't matter.  But, for users who prefer
+  to keep their devices airgapped, each roundtrip requires two manual
+  interventions---which can quickly add up to an annoying amount of work
+  when signing frequently or using multiple devices for scripted
+  multisignatures.
+
+  The disadvantage of this protocol was mentioned by Maxwell in his
+  original description, it "leaves open a [side-channel][topic side
+  channels] that has exponential cost per additional bit, via grinding
```

Comment:
```suggestion
  channels] that has an exponential cost per additional bit, via grinding
```

Reply:
Leaving as-is.  I think this is slightly more invasive a change, for no significant improvement in comprehensibility, than I'd like to make to a quote.

Reply:
Agreed, sorry, I once again failed to notice that this is inside a quote.